hardware_interface/src/fake_components/generic_system.cpp

```diff
@@ -54,6 +54,31 @@ return_type GenericSystem::configure(const hardware_interface::HardwareInfo & in
     }
   }
 
+  // Search for mimic joints
+  for (auto i = 0u; i < info_.joints.size(); ++i) {
+    const auto & joint = info_.joints.at(i);
+    if (joint.parameters.find("mimic") != joint.parameters.cend()) {
+      const auto mimicked_joint_it = std::find_if(
+        info_.joints.cbegin(), info_.joints.cend(),
```

Comment:
Is there are reason not to use "begin" and "end".

```suggestion
        info_.joints.begin(), info_.joints.end(),
```

Accoring to this [StackOverflow](https://stackoverflow.com/questions/50071833/what-is-the-differences-between-begin-end-and-cbegin-cend#50071880) (what a great reference :D) it seems there is also an const interator overload for `being` and `end` methods. So I would prefer those.